.scalafmt.conf

```diff
@@ -1,5 +1,4 @@
 version = 2.7.5
-align = more    // For pretty alignment.
+align = more // For pretty alignment.
```

Comment:
I have a question not really related to this PR, but why not to use just scalafmt?

Reply:
What do you mean, like without configuration?

Reply:
@jisantuc without scalafix I mean

Reply:
Oh! Well they do different things. The scalafmt import sort rule is substantially less powerful than the scalafix rule, which lets us control what to do with relative imports, merging imports from the same thing, coalescing to wildcard imports... The scalafmt rule we've used in the past I don't think has any knobs to turn, so it's just a sorter. Scalafix also lets us remove unused stuff automatically, both in the imports and elsewhere (well I think elsewhere -- in practice we might fail compilation first because we have warn unused and fatal warnings on).

Reply:
@jisantuc thx for clarifying!